test/integration/conftest.py

```diff
@@ -7,11 +7,10 @@
 import pytest
 import logging
 
-
 logger = logging.getLogger(__name__)
 
-
 TAG_PREFIX = 'quay.io/example/builder-test'
+KEEP_IMAGES = bool(os.environ.get('KEEP_IMAGES', False))
```

Comment:
The annoying problem with these is

```
$ KEEP_IMAGES=false python -c "import os; print(bool(os.environ.get('KEEP_IMAGES')))"
True
$ KEEP_IMAGES=0 python -c "import os; print(bool(os.environ.get('KEEP_IMAGES')))"
True
```

It'll still work for our purposes, I just wanted to point it out.

Reply:
where `distutils.util.strtobool` is the common workaround